vm/systemSmartContracts/staking.go

```diff
@@ -597,26 +597,26 @@ func (r *stakingSC) unBond(args *vmcommon.ContractCallInput) vmcommon.ReturnCode
 		return vmcommon.UserError
 	}
 	if len(registrationData.RewardAddress) == 0 {
-		r.eei.AddReturnMessage("cannot unBond a key that is not registered")
+		r.eei.AddReturnMessage(fmt.Sprintf("cannot unBond key %s that is not registered", string(args.Arguments[0])))
```

Comment:
you can define a variable `blsKey := string(args.Arguments[0])` and use it

Reply:
i will let as it is